locales/notes.ca.properties

```diff
@@ -0,0 +1,80 @@
+# General note/notebook operations
+notebooks=Blocs de Notes
+tap-anywhere-to-start=Premeu a qualsevol lloc per iniciar
+note-info=Informacio de la nota
+add-to-contacts=Afegir a Contactes
+new-notebook=Crear Bloc de Notes
+notebook-all=Totes les notes
+notebook-trash=Paperera
+notebook-action-title=Editar Bloc de notes
+notebook-action-rename=Canviar el nom
+notebook-action-delete=Eliminar
+prompt-rename-notebook=Canviar el nom del Bloc de notes:
+prompt-delete-notebook=Premeu OK per eliminar aquest Bloc de notes. \n\nOBSERVACI&Oacute;: totes les vostres notes seran enviades a la paperera.
```

Comment:
Well as i said in my first comment, i really had problems to insert latin characters in the translation file. In my case this specific message were perfectly rendered, but i had to slightly change another translations (with other words avoiding special latin characters) because i never found how to put the correct text!?

I'm not very expert in FXOS apps development, but i've contributed with at least 5 other apps and i never had this kind of issues with translations. So, as i said in my first comment, i suspect that something wrong happen with the charset management in this app.

Please, it's easier to understand me if you open for example this file:

/locales/notes.fr.properties #3

put your attention on the second word of the text "tap-anywhere-to-start".... what is it???!!! I don't know how do you see it, but i see characters like â and even an SQUARE (!!!???). It's just like you insert a latin character in a utf8 charset document but WITHOUT encode properly... it' really wrong. I think... correct me if i'm wrong.